### ISSUE TITLE
freeform.submissions not able to sort of search

### ISSUE THREAD

**Issue by StuartMcD** [solspace/craft-freeform#406]
Upgraded to the latest FF Beta 16 and Craft V4.1.4.1 on existing site that used orderby in one area and fieldsearch in another and now both have no effect on the results.

`{% set submissions = craft.freeform.submissions({
		form: registerForm,
		status: 'acceptedByPlayer',
		fieldSearch:{
			c_phone: "*"~contactNumber,
		}
		}).all() %}`

Searching for a known number with this URL: /trials/returning?search=0404+651+148 
Previously only returned the one entry and now returns ALL entries irrespective. In fact, any words, letters or characters in the search param return all entries.

In another area, have been displaying a Craft channel entry that inc. a related FF field. Was using this to sort output by firstname and `entry.teams2.ids()` was a list of the FF ID's:

Inside a Craft entry 'loop'
`
{% set playerList = craft.freeform.submissions({
  orderBy: "field_2 ASC",
  id: entry.teams2.ids()
}) %}`

ID's are honored, am using `field_2` to reference the field as the actual field label was never 'recognisied'. Now the only orderBy that works is rand() everything else doesn't change a thing. If I use `field_2` it errors.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.1.4.1
- Freeform Version: 4.0.beta-16
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade

**Additional context**

**Comment by kjmartens** [comment]
Hi @StuartMcD. I can duplicate these issues as well. We'll check into it today. :)

**Comment by kjmartens** [comment]
This is now fixed in Freeform 4.0.0-beta.18. 🙂 

The correct syntax for this would be something like:

```
{% set submissions = craft.freeform.submissions({
    form: "myFormHandle",
    orderBy: "lastName ASC, firstName ASC",
    status: ["open", "pending"],
    fieldSearch: {
      firstName: "Nacho",
    }
}).all %}
```
